### ISSUE TITLE
[Frontend] Fix the carousel section of homepage

### ISSUE THREAD

**Issue by Kajol-Kumari** [HITK-TECH-Community/Community-Website#101]
Currently the carousel section have very large image height. Fix it to better smaller sized posters content

![Screenshot from 2020-12-28 12-50-42](https://user-images.githubusercontent.com/44888949/103197105-74530d80-490b-11eb-966f-ce7b143f277d.png)

**Comment by shraddha-1402** [comment]
I would like to work on this for DWOC. Can you please assign this to me?

**Comment by codewithvk** [comment]
![Capture](https://user-imag@Kajol-Kumari s.githubusercontent.com/61119120/103198680-e842e500-490e-11eb-9cb7-c9ab2070e742.PNG)
@Kajol-Kumari Check it. If you would to like then I will made PR

**Comment by Kajol-Kumari** [comment]
@shraddha-1402 you can go ahead with this issue and @codewithvk it's a request that don't start working on any issue until you get assigned and as we are following FCFS so I am assigning this to @shraddha-1402 :)

**Comment by shraddha-1402** [comment]
![image](https://user-images.githubusercontent.com/69422250/103204133-a02abf00-491c-11eb-9170-2bd4445c508f.png)

Does this suffice your needs? Require suggestion.

**Comment by Kajol-Kumari** [comment]
yes @shraddha-1402! Just make sure it doesn't leave any blank space and the cycle continue from 1st poster, once it reaches the last one

**Comment by shraddha-1402** [comment]
> yes @shraddha-1402! Just make sure it doesn't leave any blank space and the cycle continue from 1st poster, once it reaches the last one

yes, it is the same as you are asking

**Comment by shraddha-1402** [comment]
should i try to make it responsive for mobile?

**Comment by Abhishek971999** [comment]
> should i try to make it responsive for mobile?

@shraddha-1402 Yes. Please make sure whatever designs you are making, try to make everything mobile-friendly as well.

**Comment by shraddha-1402** [comment]
> > should i try to make it responsive for mobile?
> 
> @shraddha-1402 Yes. Please make sure whatever designs you are making, try to make everything mobile-friendly as well.

okay, on it!

**Comment by shraddha-1402** [comment]
i've made the carousel responsive.
![image](https://user-images.githubusercontent.com/69422250/103273806-30810680-49e6-11eb-8e55-f62172333edd.png)
![image](https://user-images.githubusercontent.com/69422250/103273854-4d1d3e80-49e6-11eb-9769-7750205526a2.png)
![image](https://user-images.githubusercontent.com/69422250/103273994-9d949c00-49e6-11eb-967a-be4c7f9cb784.png)

**Comment by Kajol-Kumari** [comment]
@shraddha-1402 Looks good to me :)

**Comment by shraddha-1402** [comment]
> @shraddha-1402 Looks good to me :)
So should i make a pr?

**Comment by shraddha-1402** [comment]
> @shraddha-1402 Looks good to me :)

So should i make a pr?

**Comment by Kajol-Kumari** [comment]
> > @shraddha-1402 Looks good to me :)
> 
> So should i make a pr?

Yes @shraddha-1402 go ahead and make the PR

**Comment by shraddha-1402** [comment]
@Kajol-Kumari i cannot create the pr, the button is disabled
![image](https://user-images.githubusercontent.com/69422250/103331763-6cbb7200-4a8d-11eb-8488-71bed13e193c.png)
what might be wrong here, could you please help me?

**Comment by shraddha-1402** [comment]
@Kajol-Kumari @Abhishek971999  
I forked the recent snapshot of the repo and got the same issue, even without adding my code in it. 
![image](https://user-images.githubusercontent.com/69422250/103442908-2aa65200-4c80-11eb-80c5-d858da0eabf4.png)

Whereas in the previous snapshot on which I worked, the button was perfect. 
So to check where the issue might be, I compared the dependencies of both the forks and found 
```
"semantic-ui-css": "^2.4.1",
"semantic-ui-react": "^2.0.1", 
"styled-components": "^5.2.1",
```
these 3 dependencies were added.
Maybe there are some conflicts ??




To get snapshot of my commit from old fork I used:-
```
git init
git remote -v
git remote add origin https://github.com/HITK-TECH-Community/Community-Website.git
git remote -v
git fetch origin e0c0b2b0b52ec40d662e0e509d38d9e9f07435e0      
git reset --hard FETCH_HEAD
cd frontend/
npm install
npm start
```
And it works as given in this image
![image](https://user-images.githubusercontent.com/69422250/103443077-90dfa480-4c81-11eb-90a0-2b80703bd299.png)
